playwright.config.ts

```diff
@@ -73,12 +73,9 @@ export default defineConfig({
   ],
 
   /* Run your local dev server before starting the tests */
-  webServer: !!process.env.CI
-    ? undefined
-    : {
-        // only run in local environment not CI
-        command: "npm run dev",
-        url: "http://localhost:3000",
-        reuseExistingServer: !process.env.CI,
-      },
+  webServer: {
+    command: "npm run start",
+    url: "http://localhost:3000",
+    reuseExistingServer: !process.env.CI,
+  },
```

Comment:
@jbdalton one important change in this PR is that in order to run the tests you should make a production build of the app first, using the command `npm run build`. This will build the app for "prod" locally in your `dist` folder. After that, you can run `npm run playwright` which will start that prod build. This should make testing a lot easier. `npm run dev` is running a dev build which is much less performant.